app/javascript/components/supplemental_application/actions.js

```diff
@@ -2,14 +2,17 @@ import apiService from '~/apiService'
 import Alerts from '~/components/Alerts'
 import { isEmpty, find, isEqual, every, reject } from 'lodash'
 import { convertCurrency } from '~/utils/form/validations'
+import { filterChanged } from '~utils/utils'
```

Comment:
I believe so, I put that function to `app/javascript/utils/utils.js`

Reply:
right I just meant the inclusion of the `/` after the `~`

Reply:
I'm not sure what are you suggesting but if to change to `import { filterChanged } from '~utils'`, thats not going to work.

Reply:
@software-project no I'm suggesting to change to `import { filterChanged } from '~/utils/utils'` since that seems to be the dominant style (see line 4)